decidim-conferences/app/serializers/decidim/conferences/conference_serializer.rb

```diff
@@ -9,6 +9,7 @@ def serialize
         super.merge(
           {
             categories: serialize_categories,
+            taxonomies: serialize_taxonomies(resource),
```

Comment:
As we have on assemblies. 
```suggestion
            taxonomies: ,
```